mlflow/recipes/recipe.py

```diff
@@ -428,11 +425,11 @@ def __new__(cls, profile: str) -> Union[RegressionRecipe, ClassificationRecipe]:
             ) from None
 
         recipe_config = get_recipe_config(recipe_root_path=recipe_root_path, profile=profile)
-        template = recipe_config.get("template")
+        template = recipe_config.get("recipe")
```

Comment:
ditto here. Since we are removing the use of `template`, let's do it thoroughly.